Scripts/Miscellaneous/Notepad_clone/notepad.py

```diff
@@ -0,0 +1,188 @@
+from PyQt5.QtGui import *
```

Comment:
import specific package from module instead of using *
```suggestion
from PyQt5.QtGui import QVBoxLayout
```